src/serac/physics/solid_mechanics.hpp

```diff
@@ -349,6 +349,16 @@ class SolidMechanics<order, dim, Parameters<parameter_space...>, std::integer_se
    * @tparam MaterialType The solid material type
    * @tparam StateType the type that contains the internal variables for MaterialType
    * @param material A material that provides a function to evaluate stress
+   * @pre body_force must be a object that can be called with the following arguments:
+   *    1. `tensor<T,dim> x` the spatial coordinates for the quadrature point
+   *    2. `double t` the time (note: time will be handled differently in the future)
+   *    3+. `tuple{value, derivative}`, a tuple of values and derivatives for each of the trial spaces
+   *            specified in the `DependsOn<...>` argument. 
+   *
+   * @note The actual types of these arguments passed will be `double`, `tensor<double, ... >` or tuples thereof
+   *    when doing direct evaluation. When differentiating with respect to one of the inputs, its stored
+   *    values will change to `dual` numbers rather than `double`. (e.g. `tensor<double,3>` -> `tensor<dual<...>, 3>`) 
+   *
    * @param qdata the buffer of material internal variables at each quadrature point
```

Comment:
Should the prereq for `operator()` now demand that it returns Cauchy stress instead of Kirchhoff? (Note to ourselves, we need to run a spellcheck over the whole docs to look for missing h's in "Kirchhoff" 😁)

Reply:
See https://github.com/LLNL/serac/pull/853